### DISCUSSION TITLE
argoserver is failing to connect to redis

### DISCUSSION THREAD

**Discussion by midestefanis** [argoproj-labs/argocd-autopilot#266] [Q&A]
my argoserver is logging the following: "Failed to resync revoked tokens. retrying again in 1 minute: dial tcp 172.20.26.105:6379: i/o timeout"

I tried to telnet from another pod to that socket and it responds fine. I do not know what it could be

**Comment by midestefanis** [comment]
I fixed this but I don't know how to delete the question.

**Reply by CodyLeeWhite** [reply]
how! i have the same issue and im stumped

**Reply by venkatakarthikbandaru** [reply]
how did you fix this can you please let me know the solution, I am facing the same challenge @midestefanis
